Mage/src/main/java/mage/abilities/TriggeredAbilityImpl.java

```diff
@@ -71,6 +71,7 @@ public void trigger(Game game, UUID controllerId, GameEvent triggeringEvent) {
         //20091005 - 603.4
         if (checkInterveningIfClause(game)) {
             setLastTrigger(game);
+            updateTriggerCount(game);
```

Comment:
Better to use same naming for both stats methods like:
- setLastTrigger: updateTurnCounts
- updateTriggerCount: updateGameCounts